docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,12 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift exposed registry
+
+If you are using Fedora/CentOS/RHEL as host and running minishift using b2d/minikube ISO then even after following the registry xref:../openshift/openshift-docker-registry.adoc#[documents],
+you might get authentication error when try to push an image to exposed registry.
```

Comment:
I think we need to explain it little more. You should add that because of the incompatibility between the docker client of RHEL/Fedora/CentOS and Docker daemon running in Minishift Boot2Docker and Minikube ISO you might see below error if you try to push a container image to the OpenShift registry. 
```
unauthorized: authentication required
```

Reply:
"If you are using a Fedora/CentOS/RHEL host and you are running {project} with the Boot2Docker or Minikube ISO, you might see an authorization error when trying to push to the registry: ..."

Capital Minishift and when referring to the project we should use {project}. We should also add a cross-reference to the ISO section. Definitely needs a reason as well on why this is the case.